src/modules/createGame.js

```diff
@@ -0,0 +1,40 @@
+// Function to create a new game and get the unique game ID
+// async function createGame(name) {
+//   const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+//   const data = { name };
+
+//   try {
+//     const response = await fetch(url, {
+//       method: 'POST',
+//       headers: {
+//         'Content-Type': 'application/json'
+//       },
+//       body: JSON.stringify(data)
+//     });
+
+//     const result = await response.json();
+//     return result.result;
+//   } catch (error) {
+//     console.error('Error creating game:', error);
+//     return null;
+//   }
+// }
+
+// const gameName = 'Leaderboardgh7Test2';
+// createGame(gameName)
+//   .then(gameId => {
+//     if (gameId) {
+//       console.log('Game created with ID:', gameId);
+//     } else {
+//       console.log('Failed to create game.');
+//     }
+//   });
```

Comment:
- [ Optional] I recommend removing all the commented code that way your project will look amazing.